anim.js

```diff
@@ -8,9 +8,9 @@ class Anim {
   }
 
   add(to, duration = resolution, options = {}) {
-    options.easing = options.easing || 'linear';
+    options.easing = options.easing || "linear";
 
-    this.fxStack.push({ 'to': to, 'duration': duration, 'options': options });
+    this.fxStack.push({ to: to, duration: duration, options: options });
```

Comment:
Good change. I'm sure there is a linting rule to enforce this formatting. Could you add it to `eslint.rc`?